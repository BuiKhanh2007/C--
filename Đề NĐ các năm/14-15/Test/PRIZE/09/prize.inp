43345114342415114341
241121132432541215155325151351