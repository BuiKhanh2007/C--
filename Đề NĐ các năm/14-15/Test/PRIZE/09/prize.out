241121132475886329497740265692
9